### DISCUSSION TITLE
Lidl HG06492C is still available when disconnected from power

### DISCUSSION THREAD

**Discussion by FlexxFR** [Koenkk/zigbee2mqtt#6770] [Devices]
I have a Lidl HG06492C connected via Zigbee2Mqtt and it works fine. However when the power is switched off (by accident, you know by whom... ;)) the bulb remains available to the system. 
In zigbee2mqtt it's still presenting a LQI of 63.

I would expect the device would become unavailable, at least in Home Assistant. But the device appears to be online.
Meanwhile all Zigbee commands are likely to be queued by zigbee2mqtt and timeout (seen in the logs), but remain in the queue nevertheless. Until the device is powered on again, then all commands are still executed. Is this expected behavior?

When is a disconnected devices reported as unavailable? Any funny issue with LIDL/TuYA devices here?

**Comment by Koenkk** [comment]
See https://www.zigbee2mqtt.io/information/availability.html

**Reply by FlexxFR** [reply]
Thanks, this explains a lot. Never thought of searching for this functionality as other ZigBee devices did report unavailability so I believed it was a brand specific issue or something. 
However, I came to realize those ZigBee devices are connected via the IKEA tradfri plug-in.

What about the second observation/question concerning the command queueing? Is that normal behavior?
For how long are those commands retried? Or does this unavailability detection also clear the command queue?

**Reply by Koenkk** [reply]
Availablity does not influence wether commands are retried, it depends on the error but commands are tried for +- 5 times.
